.github/workflows/deploy-dev.yml

```diff
@@ -0,0 +1,44 @@
+name: Docker Build & Push and Deploy to scoop-dev
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  path-context:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Get SHA
+        id: vars
+        run: echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
+      - name: Docker Build & Push
+        uses: docker/build-push-action@v2
+        with:
+          context: ./
+          file: ./Dockerfile
+          push: true
+          tags: cornellappdev/resell-dev:${{ steps.vars.outputs.sha_short }}
+      - name: Remote SSH and Deploy
+        uses: appleboy/ssh-action@master
+        env:
+          IMAGE_TAG: ${{ steps.vars.outputs.sha_short }}
+        with:
+          host: ${{ secrets.DEV_SERVER_HOST }}
```

Comment:
maybe change the name to PROD_SERVER_HOST and PROD_SERVER_KEY since we r working w prod rn and it will make it easier to distinguish if we add a dev server in the future